components/common/navBarComponent/navBarcomponent.tsx

```diff
@@ -0,0 +1,188 @@
+import Image from "next/image";
```

Comment:
We need to add the WSO2 open source license header to these files I guess.

Reply:
Fixed